nemo_rl/models/policy/dtensor_policy_worker_v2.py

```
@@ -684,6 +686,9 @@ def train(
                             # remove flash_attn_kwargs if there are multimodal kwargs
                             if len(vlm_kwargs) > 0:
                                 del model_args["flash_attn_kwargs"]
+                            
+                            if 'flash_attn_kwargs' in model_args:
+                                del model_args['flash_attn_kwargs']
```

Ruff (0.12.2)

Use `pop` instead of `key in dict` followed by `del dict[key]`

Replace `if` statement with `.pop(..., None)`

(RUF051)